### ISSUE TITLE
local dependency - module not found

### ISSUE THREAD

**Issue by karlkauc** [beryx/badass-jlink-plugin#33]
I do have a gradle project with a dependency on a local lib - declared in the gradle dependencies as "compile files(.....)."

Every time I get an error "Module ... not found". 
If the dependency is loaded from mavenCentral ist works - if the dependency is loaded from file system it doen't work. 

I tried this for demonstration purpose with apache commons lang as a dependency. 
This one works: 

build.gradle:
```
plugins {
    id 'java'
    id 'application'
    id 'org.beryx.jlink' version '2.6.6'
    id "org.javamodularity.moduleplugin" version "1.1.1"
}

group 'com.test'
version '1.0-SNAPSHOT'

sourceCompatibility = 11
mainClassName = "com.test/com.test.Start"

repositories {
    mavenCentral()
}

dependencies {
    compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.8.1' // THIS ONE WORKS
    // compile files('commons-lang3-3.8.1.jar') // THIS ONE DO NOT WORK
}

jlink {
    options = ['--strip-debug', '--compress', '2', '--no-header-files', '--no-man-pages']
    launcher {
        name = 'jlinkTest'
    }
}
```

module-info.java:
```
module com.test {
    requires org.apache.commons.lang3;
}
```

com\test\Start.java:
```
package com.test;

import org.apache.commons.lang3.RandomStringUtils;

public class Start {

    public static void main(String[] args) {
        System.out.println(RandomStringUtils.randomAlphabetic(5));
    }
}
```
If i switch the dependency from "compile group" to "compile files" I can run it with gradle - but not jlink it.

**Comment by siordache** [comment]
This is a current limitation of the plugin. I will add support for local dependencies in the next release.

**Comment by karlkauc** [comment]
Thank you.

**Comment by siordache** [comment]
Fixed in release 2.7.0.
